frontend/src/prompted_readings/analysis_view.js

```diff
@@ -1,4 +1,34 @@
 import React from "react";
+import PropTypes from "prop-types";
+
+class SentimentScores extends React.Component {
+
+    constructor(props) {
```

Comment:
Setting state here is redundant -- you can access props directly. This component doesn't actually need to implement a constructor, as the parent class constructor is enough.